opencensus/plugins/grpc/internal/filter.h

```diff
@@ -59,8 +59,12 @@ class CensusContext {
     return 0;
   }
 
-  void RecordSentMessageEvent(uint32_t id, uint32_t size);
-  void RecordReceivedMessageEvent(uint32_t id, uint32_t size);
+  void RecordSentMessageEvent(uint32_t id, uint32_t size) {
```

Comment:
Why not open-code these at the call site since Span() is public anyway.

Reply:
Done.